CHANGELOG.rst

```diff
@@ -27,8 +27,10 @@ Added
 - New EnlargeWithAncilla pass for adding ancilla qubits after a Layout
   selection pass (#1603).
 - New Unroll2Q pass for unrolling gates down to just 1q or 2q gates (#1614).
+- Added a RunConfig object for configurations for run configurations to be used
+  in compile and circuits_to_qobj. (#1629)
 - Added support for register slicing when applying operations to a register (#1643).
-- Added in new parameter ``justify`` to the text circuit drawer to say how the
+- Added in new parameter ``justify`` to the text, mpl and latex circuit drawers to say how the
   circuit should be aligned. (#1725)
```

Comment:
I'd probably add the 3 PRs here for each backend since 1725 didn't add it to all the backends.

Reply:
I have done